ROADMAP.md

```diff
@@ -109,31 +109,37 @@ Halves are defined as:
 **Checking points for releasing V1-RC**
 
 * **Validator**
-    - Validators for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, and Corda
+    - Validators for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, Go-Ethereum, and Corda
 
 * **Verifier**
     - All sync/async requests from BLP must communicate with  Validator (toward ledgers) via Verifier
-    - transaction signer features for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, and Corda
+    - transaction signer features for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, Go-Ethereum, and Corda
 
 * **BLP-attached optional plugins**
 
 * **BLP applications**
-    - BLP applications using each ledger of Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, and Corda
+    - BLP applications using each ledger of Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, Go-Ethereum, and Corda
 
 * **Test ledger tools**
-    - Ledger tools for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, and Corda
+    - Ledger tools for Hyperledger ledgers (Fabric, Sawtooth, Besu, Indy, Iroha), Quorum, Go-Ethereum, and Corda
 
 * **Service API and Admin API**
     - SDK for BLP
     - SDK for LedgerPlugin
 
 * **Support tools**
 
+* **Dockernize**
```

Comment:
@takeutak Typo?

Reply:
@takeutak +1

Reply:
Sorry, I overlooked it. I will change it soon.